tools/codegen/proc_interface_gen/main.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017 Istio Authors
```

Comment:
I'd prefer the directory structure mirror the main mixer repo's, with everything as a library and a cmd package with this kind of thing in it. `tools/codegen/cmd/...`